src/screen.rs

```diff
@@ -9,53 +9,61 @@ use termion;
 use termion::clear;
 use termion::cursor;
 use termion::input::MouseTerminal;
+// use termion::screen::AlternateScreen;
 use termion::raw::IntoRawMode;
 use termion::raw::RawTerminal;
 
 use serde_json;
 
 use core::Core;
 use view::View;
+use errors::*;
 
 pub struct Screen {
+    //pub stdout: MouseTerminal<AlternateScreen<RawTerminal<std::io::Stdout>>>,
```

Comment:
Actually we _want_ the alternate `AlternateScreen` version. I commented it out temporarily, because I could not see the stacktrace on panic. I don't know why the stacktrace got printed in the alternate screen buffer though. That's annoying because we do want to see where we panic when that happens.